python/lsst/verify/measurement.py

```diff
@@ -193,8 +193,12 @@ def quantity(self):
     def quantity(self, q):
         # a quantity can be None or a Quantity
         if not isinstance(q, u.Quantity) and q is not None:
-            message = '{0} is not an astropy.units.Quantity-type'
-            raise TypeError(message.format(q))
+            try:
+                q = q*u.dimensionless_unscaled
+            except (TypeError, ValueError):
+                message = '{0} cannot be coerced into an ' + \
```

Comment:
I like doing this for multiline strings:

```
('line one'
 'line two)
```

Reply:
Thinking out loud here; if we move away from `None` as the default, we might _not_ want to accept `q` as None here. Keep this in mind with that DM-11253.

Reply:
OK I think we are good since we decided to allow `None` to survive as the default.